pinot-plugins/pinot-batch-ingestion/v0_deprecated/pinot-hadoop/src/test/java/org/apache/pinot/integration/tests/HadoopSegmentBuildPushOfflineClusterIntegrationTest.java

```diff
@@ -1,248 +0,0 @@
-/**
```

Comment:
Why does this test get removed?

Reply:
This is already disabled long time ago and is not working properly.
Adding hadoop-minicluster dependency might cause lots of dependency issue. I don't think it is worth it just for this disabled test.